spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -781,9 +781,10 @@ def _run(self, run_time, run_until_complete=False):
         """
         self.verify_not_running()
         if (self._has_ran and
-                self._executable_start_type not in [
-                    ExecutableStartType.USES_SIMULATION_INTERFACE,
-                    ExecutableStartType.NO_APPLICATION]):
+                ExecutableType.USES_SIMULATION_INTERFACE not in
+                self._executable_types and
+                ExecutableType.NO_APPLICATION not in
+                self._executable_types):
```

Comment:
This could do to be updated - you can reuse supports auto pause and resume I think though...

Reply:
very true. fixing

Reply:
fixed